site/src/scss/_homepage.scss

```diff
@@ -113,7 +113,10 @@
   }
   @include breakpoint(medium) {
     .shot {display: block}
-    .note {margin-top: rem-calc(93);}
+    .note {
+      margin-top: rem-calc(93);
+      margin-left: rem-calc(70);
```

Comment:
Hmm, I'd prefer to handle this by shrinking the size of the xy-grid cell on medium+.  Actually, given that @jaymejohnson is currently refactoring the About pane for the image (.shot), how about if we exclude that part from this PR for now and revisit once his work is done?

Reply:
Works for me.